packages/core/styles/_icons.scss

```diff
@@ -75,8 +73,7 @@
   }
 }
 
-/* Icon size adjustments
-   ========================================================================== */
+/* Icon size adjustments */
```

Comment:
Should this be // like the 2 other comments above for defaults, or should they both be /* like this one?

Reply:
Updated!